homeassistant/components/slide/cover.py

```diff
@@ -55,19 +60,19 @@ def __init__(self, api, slide):
     @property
     def is_opening(self) -> bool:
         """Return if the cover is opening or not."""
-        return self._slide["state"] == STATE_OPENING
+        return self._slide["state"] == CoverState.OPENING
```

Comment:
This integration is abusing our entity state constants for its own device state calculations. Please refactor this separetely to use integration specific constants.